# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/style/moz.build
# Commit: 3d30a8d83d77
# Full Hash: 3d30a8d83d77cfb25c9d75aae2daa01051810ff0
# Author: Andrew Osmond <aosmond@mozilla.com>
# Date: 2022-06-29 03:44:30
# Regressor Bug: 1771493
# File Overlap Count: 1
# Description:
#   Bug 1771493 - Part 1. Split FontFace(Set) into DOM facing and implementation classes. r=emilio
#   
#   This patch aims to split the implementation details of FontFace(Set)
#   into FontFace(Set)Impl classes. The Impl variants have threadsafe
#   refcounting and are intended to be the basis for worker support for
# ==============================================================================

diff -r 53c490e52eb6 -r 3d30a8d83d77 layout/style/moz.build
--- a/layout/style/moz.build	Tue Jun 28 21:23:39 2022 +0000
+++ b/layout/style/moz.build	Tue Jun 28 21:46:59 2022 +0000
@@ -140,7 +140,9 @@
     "CSSSupportsRule.h",
     "CSSValue.h",
     "FontFace.h",
+    "FontFaceImpl.h",
     "FontFaceSet.h",
+    "FontFaceSetImpl.h",
     "FontFaceSetIterator.h",
     "MediaList.h",
     "MediaQueryList.h",
@@ -188,7 +190,9 @@
     "DocumentStyleRootIterator.cpp",
     "ErrorReporter.cpp",
     "FontFace.cpp",
+    "FontFaceImpl.cpp",
     "FontFaceSet.cpp",
+    "FontFaceSetImpl.cpp",
     "FontFaceSetIterator.cpp",
     "FontPreloader.cpp",
     "GeckoBindings.cpp",